vaccine_feed_ingest/runners/ct/state/normalize.py

```diff
@@ -0,0 +1,207 @@
+
+    for field in ("description", "application_process", "hours_of_operation"):
+        value = site[field].lower()
+        if "appointment" in value:
+            appointments = True
+        if re.search("walk[ |-]in", value):
+            drop_in = True
+
+    if drop_in or appointments:
+        return schema.Availability(drop_in=drop_in, appointments=appointments)
+    else:
+        return None
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+
+    """
+    "site_phones": {
+        "phone_1": "860-827-7690",
+        "phone_2": "833-943-5721",
+        "phone_1_name": "Voice",
+        "phone_2_name": "Voice",
+        "phone_1_description": "Hartford HealthCare Access Center (M-F: 8am-5pm)",
+        "phone_2_description": "Hartford HealthCare Access Center (M-F: 8am-5pm)",
+        "phone_toll_free_description": ""
+    },
+    """
+    phones = [k for k in site["site_phones"] if re.match(r"phone_\d+", k)]
+    for phone in phones:
+        try:
+            contacts.append(
+                schema.Contact(
+                    phone=site["site_phones"][phone],
+                )
+            )
+        except ValidationError:
+            logger.debug("Invalid phone {site['site_phones'][phone]}")
+
+    if site.get("agency_phones", {}).get("phone_tty", None):
+        contacts.append(
+            schema.Contact(
+                other=f"{site['agency_phones']['phone_tty']} TTY",
+            )
+        )
+
+    # the site and agency website are frequently the same, dedupe 'em
+    websites = set()
+    for key in ("agency_website", "site_website"):
+        if site[key]:
+            websites.add(site[key])
+    for website in websites:
+        if not website.startswith("http"):
+            website = f"https://{website}"
+        try:
+            contacts.append(
+                schema.Contact(
+                    website=website,
+                )
+            )
+        except ValidationError:
+            logger.debug("Invalid website {website}")
+
+    return contacts or None
+
+
+def _get_id(site: dict) -> str:
+    return f"ct_state:{site['number']}"
+
+
+def _get_languages(site: dict) -> Optional[List[str]]:
+    potentials = {
+        "spanish": "es",
+        "french": "fr",
+        "polish": "pl",
+    }
+
+    languages = []
+    for lang in re.split(r"\s*,\s*", site.get("languages") or ""):
+        if lang.lower() in potentials:
+            languages.append(potentials[lang.lower()])
+    return languages or None
+
+
+def _get_location(site: dict) -> Optional[schema.LatLng]:
+    float_pattern = r"-?\d+\.\d+"
+    match = re.search(
+        f"(?P<lng>{float_pattern}) (?P<lat>{float_pattern})", site["location"]
+    )
+    if match:
+        """
+        "POINT (-73.04276 41.55975)"
+        """
+        return schema.LatLng(
+            latitude=float(match.group("lat")),
+            longitude=float(match.group("lng")),
+        )
+    else:
+        return None
```

Comment:
In the follow-up: should we log the location when we fail to parse a lat/lng? It's challenging to use this data downstream without those.